R/utils-shiny.R

```diff
@@ -64,3 +64,15 @@ anyNamed <- function(x) {
   }
   any(nzchar(nms))
 }
+
+
+# Copy of shiny:::shinyInputLabel()
+shiny_input_label <- function(id, label = NULL) {
+  tags$label(
+    label,
+    class = "control-label",
+    class = if (is.null(label)) "shiny-label-null",
+    id = paste0(id, "-label"),
+    `for` = id
+  )
+}
```

Comment:
If we're only needing to duplicate the logic here, maybe just reach into shiny's namespace. There's such a tiny chance we'd remember to update this function if we make changes in shiny.

```suggestion
shiny_input_label <- function(id, label = NULL) {
  asNamespace("shiny")[["shinyInputLabel"]](id, label)
}
```